lambda/MAS/Services/ContentService.cs

```diff
@@ -14,13 +16,28 @@ public interface IContentService
 
     public class ContentService : IContentService
     {
+        private readonly ILogger<ContentController> _logger;
+        public ContentService(ILogger<ContentController> logger)
+        {
+            _logger = logger;
+        }
         public async Task<Item> GetItemAsync(string itemId)
         {
             using (WebClient client = new WebClient())
             {
-                var jsonStr = await client.DownloadStringTaskAsync(new Uri(AppSettings.CMSConfig.URI + itemId));
-                Item json = JsonConvert.DeserializeObject<Item>(jsonStr);
-                return json;
+                var json = await client.DownloadStringTaskAsync(new Uri(AppSettings.CMSConfig.URI + itemId));
+                Item item = null;
+                try
+                {
+                    item = JsonConvert.DeserializeObject<Item>(json);
+                }
+                catch(Exception e)
+                {
+                    _logger.Log(LogLevel.Error, e.Message);
+                    throw e;
```

Comment:
Do we still need to throw the error if we're catching it?

Reply:
I think so that way it'll bomb out when debugging locally. On live this will be handled gracefully by a 500 error page.